go.mod

```diff
@@ -104,6 +104,7 @@ require (
 	github.com/bufbuild/protocompile v0.4.0 // indirect
 	github.com/c-bata/go-prompt v0.2.6 // indirect
 	github.com/cenkalti/backoff v2.2.1+incompatible // indirect
+	github.com/confluentinc/ccloud-sdk-go-v2/billing v0.3.0
```

Comment:
did you run `go mod tidy`? This should be grouped together with other packages in ccloud-sdk-go-v2

Reply:
oh i just noticed there's two separate require sections, which is why go mod tidy didn't re-order